mne_bids/tests/test_read.py

```diff
@@ -295,19 +295,60 @@ def test_handle_coords_reading():
     write_raw_bids(raw, bids_basename, bids_root, overwrite=True)
 
     # read in the data and assert montage is the same
+    # regardless of 'm', 'cm', 'mm', or 'pixel'
     bids_fname = bids_basename + "_ieeg.edf"
-    raw_test = read_raw_bids(bids_fname, bids_root)
-
-    # obtain the sensor positions
-    orig_locs = raw.info['dig'][1]
-    test_locs = raw_test.info['dig'][1]
-    assert orig_locs == test_locs
-    assert not object_diff(raw.info['chs'], raw_test.info['chs'])
```

Comment:
why killing an existing test? this suggests it was working before but you had to remove the test.